routing/routing_integration_tests/turn_test.cpp

```diff
@@ -809,3 +797,51 @@ UNIT_TEST(BelorussiaMinskTest)
   integration::TestTurnCount(route, 1 /* expectedTurnCount */);
   integration::GetNthTurn(route, 0).TestValid().TestDirection(CarDirection::TurnRight);
 }
+
+UNIT_TEST(EnglandLondonExitToLeftTest)
+{
+  TRouteResult const routeResult =
+      integration::CalculateRoute(integration::GetVehicleComponents<VehicleType::Car>(),
+                                  MercatorBounds::FromLatLon(51.603582, 0.266995), {0., 0.},
+                                  MercatorBounds::FromLatLon(51.606785, 0.264055));
+
+  Route const & route = *routeResult.first;
+  IRouter::ResultCode const result = routeResult.second;
+
+  TEST_EQUAL(result, IRouter::NoError, ());
+  integration::TestTurnCount(route, 1 /* expectedTurnCount */);
+  integration::GetNthTurn(route, 0).TestValid().TestDirection(CarDirection::ExitHighwayToLeft);
+}
+
+// Test on the route from Leninsky prospect to its stand-in road and turns generated on the route.
```

Comment:
https://en.wikipedia.org/wiki/Frontage_road ?

Reply:
Thanks. Let's call it "frontage road".